pantheon-systems/drops-8
# Update to Drupal 8.0.0-rc3.

For more information, see https://www.drupal.org/node/2608078

Instructions to test:

- Create a new Drupal 8 site on Pantheon; start with Drupal 8.0.0-rc1.
- When site creation is finished, visit dashboard.
- Switch to "git" mode.
- Clone your site locally.
- Apply the files from this PR on top of your local checkout.
  - git remote add drops-8 git@github.com:pantheon-systems/drops-8.git
  - git fetch drops-8
  - git merge drops-8/update-8.0.0-rc3
- Push your files back up to Pantheon.
- Switch back to sftp mode.
- Visit your site. Step through the installation process. Look for any problems.

I also have some Behat tests that are not quite hooked in to drops-8. See [pantheon-systems/updates-8](https://github.com/pantheon-systems/updates-8). n.b. this repository is not updated to Drupal 8.0.0-rc3; this is for academic interest only.

Note that the "minimal" installation profile is still broken on Pantheon (c.f. #96), so please use the standard installation profile to test.

See also #106 and #107.
